uk.co.compucorp.civicrm.hrleaveandabsences/views/manager-leave/components/manager-leave-requests.html

```diff
@@ -118,12 +118,22 @@
                     <i class="fa fa-ellipsis-v"></i>
                   </a>
                   <ul class="dropdown-menu dropdown-menu-right" uib-dropdown-menu>
-                    <li ng-repeat="action in ctrl.actionsFor(request)">
-                      <a ng-if="action === 'cancel'" href
+                    <li ng-repeat="action in ctrl.actionsFor(request)" ng-switch on="action">
+                      <a ng-switch-when="cancel" href
                         ng-click="ctrl.action(request, action)">
                         {{action}}
                       </a>
-                      <a ng-if="action !== 'cancel'" href
+                      <a ng-switch-when="approve" href
+                        ng-click="ctrl.action(request, action)">
+                        {{action}}
+                      </a>
+                      <a ng-switch-when="reject" href
+                        ng-click="ctrl.action(request, action)">
+                        {{action}}
+                      </a>
```

Comment:
I could do this if it was Angular 1.5.10. 
<a ng-switch-when="reject|approve|cancel" href
  ng-click="ctrl.action(request, action)">
  {{action}}
</a>
Since Angular does not have if-else like statements, I will have to find an alternative way to concise the code.